lib/chef/knife/vsphere_vm_find.rb

```diff
@@ -189,6 +193,10 @@ def run
        if get_config(:ip)
          print "#{ui.color("IP:", :cyan)} #{vmc.guest.ipAddress}\t"
        end
+       if get_config(:ips)
+         networks = vmc.guest.net.map { |net| net.network + ":" + net.ipAddress.select { |i| i[/^(?:[0-9]{1,3}\.){3}[0-9]{1,3}$/] }[0] }.join(",")
```

Comment:
Yes could definitely do that. will update the PR shortly, thanks for the suggestion!